color/p5js/Law of simmilarity + color/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: [p5js]
+author: [Daniele Maselli]
+author-url: https://github.com/DanieleMaselli
+title: color + similarity
```

Comment:
Please give it a meaningful name. We can't have the `+` in the Title. Name it „5-dots“ or something more creative.